MatrixSDK/Crypto/MXCrypto.m

```diff
@@ -1398,7 +1398,7 @@ - (instancetype)initWithMatrixSession:(MXSession*)matrixSession cryptoQueue:(dis
 
         oneTimeKeyCount = -1;
 
-        _backup = [[MXKeyBackup alloc] initWithMatrixSession:_mxSession];
+        _backup = [[MXKeyBackup alloc] initWithCrypto:self];
```

Comment:
As mentioned above a retain cycle is created as MXKeyBackup has a strong reference to MXCrypto and MXCrypto has a strong reference to MXKeyBackup.